src/components/fetch-wrapper.jsx

```diff
@@ -86,6 +69,23 @@ function withFetching(WrappedComponent, props) {
         this.xhr.abort();
       }
     }
+
+    render() {
```

Comment:
what's the reason for the change in this file?

Reply:
This was a lint error causing `npm start` to fail, I have removed this change and make a different PR for it.